glide-core/src/client/standalone_client.rs

```diff
@@ -241,15 +253,53 @@ impl StandaloneClient {
                 let _ = latest_read_replica_index.compare_exchange_weak(
                     initial_index,
                     index,
-                    std::sync::atomic::Ordering::Relaxed,
-                    std::sync::atomic::Ordering::Relaxed,
+                    Ordering::Relaxed,
+                    Ordering::Relaxed,
                 );
                 return connection;
             }
         }
     }
 
-    fn get_connection(&self, readonly: bool) -> &ReconnectingConnection {
+    async fn round_robin_read_from_replica_az_awareness(
```

Comment:
I am worried that if we fix a bug in this method, we will also need to fix the one that is a copy - paste of this one above

Reply:
what is the bug?